packages/core/src/components/tabs/tabs.tsx

```diff
@@ -265,15 +339,40 @@ export class Tabs extends AbstractPureComponent<ITabsProps, ITabsState> {
 
         const tabIdSelector = `${TAB_SELECTOR}[data-tab-id="${this.state.selectedTabId}"]`;
         const selectedTabElement = this.tablistElement.querySelector(tabIdSelector) as HTMLElement;
+        // Check if there are any overflow elementsR
+        const isOverflowing = this.overflowElement && this.overflowElement.state.overflow.length;
```

Comment:
🙅‍♂️ we must be able to do this without reaching _into_ another component's internals. if it's truly unavoidable then we should provide a hook via a new prop. 

treat the insides of other components as black boxes, much like you treat your friends 😄

Reply:
It might be unavoidable, I think state has been updated but the actually element has not rerendered to reflect that.

Reply:
I suppose we can grab it using querySelector on the tablist element, either tab or a public getter method? I'll make the changes you suggested here and see if it works with the querySelector